### ISSUE TITLE
`wrapText` with width `1` hangs indefinetly when hyphenation is turned on

### ISSUE THREAD

**Issue by lauft** [GothenburgBitFactory/libshared#24]
This is somehow related to #23 and https://github.com/GothenburgBitFactory/timewarrior/issues/309.

I noticed that when setting the `width` parameter of `wrapText(...)` to `1` it fails (hangs indefinetly) when hyphenation is turned on:
```
  text = "one";
  lines.clear ();
  wrapText (lines, text, 1, true);
```
Without hyphenation, it passes:
```
  text = "one";
  lines.clear ();
  wrapText (lines, text, 1, false);
  t.is (lines.size (), (size_t) 3, "wrapText 'one' -> 3 lines");
  t.is (lines[0], "o", "wrapText line 0 -> 'o'");
  t.is (lines[1], "n", "wrapText line 1 -> 'n'");
  t.is (lines[2], "e", "wrapText line 1 -> 'e'");
```

**Comment by wodny** [comment]
Hi, please check if the patch from the PR is good enough.

**Comment by lauft** [comment]
Fixed with #28
